### ISSUE TITLE
Feature request - Finetuning or Pretraining for Urdu

### ISSUE THREAD

**Issue by hunzlausman** [snakers4/silero-vad#414]
Please tell me if I can fine-tune this model on urdu dataset or train from scratch with same architecture for Urdu??
In short please elaborate the architecture of this model and how it is so fast.

**Comment by snakers4** [comment]
This year we will probably share a fine-tuning recipe

**Comment by hunzlausman** [comment]
Thank you so much for the reply Alexander.Can you please tell me the
architecture it is based on or how you achieved this perfection in speed
and accuracy both at the same time?And also the expected time in days or
months for a new release you're talking about as I'm greatly impressed by
and looking for this over last few months so can't wait more.If you
elaborate the architecture, I can contribute alot in your project.

**Comment by hunzlausman** [comment]
Thank you so much for the reply can you please tell me the name of
archtecture it is based on?And expected time in days or months for the new
release you're talking about as I am greatly impressed by accuracy and
speed together in it.I was looking for this over last few months cant wait
more please just elaborate the architecture i will try to make and
contribute with you.

**Comment by filtercodes** [comment]
Any news on this??? For me this model turned out very much useless without fine-tuning, because I get too much false positives and that makes the system unstable. Will stick to other options until the fine-tuning recipe is available.

**Comment by hunzlausman** [comment]
No updates yet. Exactly, any model like this is worthless without having
control of finetuning or modifications.

**Comment by snakers4** [comment]
As a first step - we released the dataset - https://github.com/snakers4/silero-vad/tree/master/datasets

**Comment by markjosims** [comment]
This is excting news, I'm looking forward to the finetuning scripts being released

**Comment by snakers4** [comment]
The new VAD version was released just now - https://github.com/snakers4/silero-vad/issues/2#issuecomment-2195433115

It supports more than 6,000 languages now

Fine-tuning code will be released soon

**Comment by hunzlausman** [comment]
Please we need for speech to text not vad.

**Comment by filtercodes** [comment]
> The new VAD version was released just now - [#2 (comment)](https://github.com/snakers4/silero-vad/issues/2#issuecomment-2195433115)
> 
> It supports more than 6,000 languages now
> 
> Fine-tuning code will be released soon

Looking forward to try it out!

**Comment by filtercodes** [comment]
Yes, it doesn't work with the python script. I'm having this error:


<img width="597" alt="Screenshot 2024-06-28 at 09 31 49" src="https://github.com/snakers4/silero-vad/assets/15360026/b09da098-5a01-46a3-b790-07c09501b277">

**Comment by gibsonpil** [comment]
> Do we have any update on the finetuning version?

It seems that they released the code, only trouble is that it's in Russian. Nothing Google Translate can't work around though.
https://github.com/snakers4/silero-vad/tree/master/tuning

**Comment by hunzlausman** [comment]
Nope, we demand for ASR, they just released for VAD.

**Comment by gibsonpil** [comment]
This is the VAD repo. If you are trying to get finetuning code for ASR open an issue in that repository.
